pom.xml

```diff
@@ -499,8 +526,8 @@ org.netbeans.html.boot.impl:org.netbeans.html.boot.fx:org.netbeans.html.context.
                           </execution>
                       </executions>
                       <configuration>
-                          <packages>${publicPackages}</packages>
-                          <releaseVersion>1.6</releaseVersion>
+                          <packages>${sigtestPackages}</packages>
```

Comment:
This is unfortunate, as the `.sigtest` artifact isn't generated for the new projects.